history/subscriber/persister.py

```diff
@@ -268,33 +268,24 @@ def on_put(req, resp):
                 'Logging level must be DEBUG, INFO, WARNING, ERROR or CRITICAL!', 'level')
 
 
-def handle_choose_service(config, persister, check):
+def start_dojot_messenger(config, persister):
 
     messenger = Messenger("Persister", config)
     messenger.init()
-
-    messenger.create_channel(config.dojot['subjects']['devices'], "r")
-    messenger.create_channel(config.dojot['subjects']['device_data'], "r")
-
-    if conf.dojot_notification_on != True:
-        LOGGER.info("all services")
-
+    # Persister Only Notification
+    messenger.create_channel("dojot.notifications", "r")
+    messenger.on(config.dojot['subjects']['tenancy'],"message", persister.handle_new_tenant)
+    messenger.on("dojot.notifications", "message",persister.handle_notification)
+    LOGGER.info('Only notifications are enabled')
```

Comment:
Note that this log message doesn't make sense once the the devices' events can be persisted if  dojot_persist_notifications_only is false.

Reply:
Okay, I will remove